jsx/Form.js

```diff
@@ -2233,7 +2233,7 @@ class RadioElement extends React.Component {
 }
 RadioElement.propTypes = {
   name: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
+  label: PropTypes.string,
```

Comment:
This looks like it is, in fact, required in the render method.

@HenriRabalais @zaliqarosli @maltheism ? (I'm not sure who wrote this particular component..)

Reply:
@driusan I was the one that made the RadioElement component and the label isn't required for functionality. Specifically, if the developer doesn't supply a "label" the radio options will be visible without a label. The redesigned DQT uses the RadioElement component without a label and for switching between Cross-sectional and Longitudinal. It looks ugly if a label is supplied.

Reply:
But the render function unconditionally uses the label prop on line 2223

Reply:
@driusan nothing will be rendered on line 2223 if this.props.label is null